google/_modules/gke/service_account.tf

```diff
@@ -1,5 +1,5 @@
 resource "google_service_account" "current" {
-  account_id = var.metadata_name
+  account_id = substr(var.metadata_name, 0, 30)
```

Comment:
Looks good in general, one question, why not `0, 27`? Doesn't the regex say between 4 and 28 characters? I may be reading the regex wrong.

Reply:
Yeah I also read it like that but if you actually try it its 30 characters :man_shrugging: I tested it manually and also tried a regex checker :laughing: